src/main/java/de/retest/recheck/cli/FilterUtil.java

```diff
@@ -27,4 +34,22 @@ public static Filter getExcludeFilterFiles( final List<String> exclude ) {
 		return Stream.concat( excluded, Stream.of( RecheckIgnoreUtil.loadRecheckIgnore() ) ) //
 				.collect( Collectors.collectingAndThen( Collectors.toList(), CompoundFilter::new ) );
 	}
+
+	public static List<String> getInvalidFilters( final List<String> exclude ) {
+		final List<String> invalidFilterFiles = new ArrayList<>();
+		if ( exclude == null ) {
+			return invalidFilterFiles;
+		}
+		for ( final String excludeFilterName : exclude ) {
+			if ( !SearchFilterFiles.toFileNameFilterMapping().containsKey( excludeFilterName ) ) {
+				invalidFilterFiles.add( excludeFilterName );
+			}
+		}
+		return invalidFilterFiles;
+	}
+
+	public static void logWarningForInvalidFilters( final List<String> exclude ) {
```

Comment:
As the list only contains invalid filters (filter names), the `exclude` parameter should be named accordingly.